ebu_tt_live/bindings/converters/ebutt3_ebuttd.py

```diff
@@ -267,6 +268,12 @@ def convert_div(self, div_in, dataset):
         return new_elem
 
     def convert_p(self, p_in, dataset):
+        #added to remove timings when timings are present on child elements
+        if p_in.is_timed_leaf() == False:
+            for span in p_in.span:
+                if span.begin and span.end:
```

Comment:
What would happen if _only_ `begin` _or_ `end` attributes were to exist on a `p` element? Is that even possible?

Reply:
It sure is possible. The answer is that if either `begin` or `end` is on a `p` and either `begin` or `end` is on a child `span`, all the timings should be moved to all the child spans.